<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crazyflie Nano Quadcopter Control</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="quickConnectButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quickconnect using the last connection parameters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Quick Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QProgressBar" name="linkQualityBar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="format">
           <string>Link quality %p%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="connectButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Connect to a Crazyflie&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QProgressBar" name="batteryBar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>3000</number>
          </property>
          <property name="maximum">
           <number>4200</number>
          </property>
          <property name="value">
           <number>3000</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
          <property name="format">
           <string>Battery %v mV</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="autoReconnectCheckBox">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Auto Reconnect</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menuItemExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="tabsMenuItem"/>
    <addaction name="toolboxesMenuItem"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuItemAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="logConfigAction"/>
    <addaction name="separator"/>
    <addaction name="_menuItem_openconfigfolder"/>
   </widget>
   <widget class="QMenu" name="menuCrazyflie">
    <property name="title">
     <string>Crazyflie</string>
    </property>
    <addaction name="menuItemConnect"/>
    <addaction name="menuItemQuickConnect"/>
    <addaction name="menuItemBootloader"/>
    <addaction name="_menu_cf2_config"/>
    <addaction name="_menu_cf1_config"/>
   </widget>
   <widget class="QMenu" name="_menu_inputdevice">
    <property name="title">
     <string>Input device</string>
    </property>
    <widget class="QMenu" name="_menu_mappings">
     <property name="title">
      <string>Mappings</string>
     </property>
    </widget>
    <widget class="QMenu" name="_menu_devices">
     <property name="title">
      <string>Devices</string>
     </property>
    </widget>
    <widget class="QMenu" name="_menu_mux">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Mux</string>
     </property>
    </widget>
    <addaction name="menuItemConfInputDevice"/>
    <addaction name="_menu_devices"/>
    <addaction name="_menu_mappings"/>
    <addaction name="_menu_mux"/>
    <addaction name="_menuitem_rescandevices"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCrazyflie"/>
   <addaction name="_menu_inputdevice"/>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuItemConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="menuItemExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="menuItemRegulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Regulation</string>
   </property>
  </action>
  <action name="menuItemPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plotting</string>
   </property>
  </action>
  <action name="menuItemFlightdata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flight data</string>
   </property>
  </action>
  <action name="menuItemService">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Service</string>
   </property>
  </action>
  <action name="menuItemAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="menuItemQuickConnect">
   <property name="text">
    <string>Quick connect</string>
   </property>
  </action>
  <action name="menuItemConfInputDevice">
   <property name="text">
    <string>Configure device mapping</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSelect_input_device">
   <property name="text">
    <string>Select input device</string>
   </property>
  </action>
  <action name="menuItemConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="toolboxesMenuItem">
   <property name="text">
    <string>Toolboxes</string>
   </property>
  </action>
  <action name="tabsMenuItem">
   <property name="text">
    <string>Tabs</string>
   </property>
  </action>
  <action name="logConfigAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logging configurations</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="menuItemBootloader">
   <property name="text">
    <string>Bootloader</string>
   </property>
  </action>
  <action name="actionAsdfasdf">
   <property name="text">
    <string>asdfasdf</string>
   </property>
  </action>
  <action name="actionSdfasdf">
   <property name="text">
    <string>sdfasdf</string>
   </property>
  </action>
  <action name="_menuitem_rescandevices">
   <property name="text">
    <string>Rescan devices</string>
   </property>
  </action>
  <action name="_menuItem_openconfigfolder">
   <property name="text">
    <string>Open config folder</string>
   </property>
  </action>
  <action name="_menu_cf2_config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Configure 2.0</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="_menu_cf_config_2">
   <property name="text">
    <string>Configure 1.0</string>
   </property>
  </action>
  <action name="_menu_cf1_config">
   <property name="text">
    <string>Configure 1.0</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
